### ISSUE TITLE
Add accessibility features

### ISSUE THREAD

**Issue by stevenbenner** [stevenbenner/jquery-powertip#3]
Currently tooltips are not accessible to visually impaired users or users who navigate with the keyboard. Accessibility features should be added to make tooltips useful to those users.
- It has been suggested that PowerTip implement [ARIA](http://webaim.org/techniques/aria/) standards ([intro](http://www.w3.org/WAI/intro/aria), [spec](http://www.w3.org/TR/wai-aria/)) similar to [this example](http://test.cita.uiuc.edu/aria/tooltip/tooltip1.php).
- Tooltips should open on focus and close on blur to support keyboard navigation.

**Comment by speedplane** [comment]
I know it's a decade old, but accessibility features are more important now than ever. If there is any way I can help, would be happy to contribute.

**Comment by stevenbenner** [comment]
I would love to see screen reader support added.

It's actually quite embarrassing that it never got implemented, as it was something I had planned to add from the beginning. When I first tried to do this, the tooling implementing ARIA was so terrible to work with, the behavior so seemingly random, and the effects so inconsistent between screen reader programs. I just gave up after a while. It seemed that the requirements of PowerTip were incompatible with screen reader software implementing ARIA.

I hope it is better now, but I haven't been following the technology much in recent years. On Windows, it seems that NVDA has gained more traction, and JAWS less, which is good news if my previous experience still holds true.

If you're volunteering to try to get AIRA working, then I would gratefully accept. But I do have a couple requests:

 1. If it looks like a big project then break it up into small pull requests with small changes and submit them as you make progress, rather than as one big collection of commits (so it's easy for me to code review and asses risk along the way).
 2. If it will require breaking backwards compatibility (like the unique `id` problem mentioned above), please confirm with me before putting any time into it (because don't have plans for a new major version in the foreseeable future).

Right now PowerTip is basically in maintenance mode, since interest has dried up in favor of newer tech. So I want to keep changes low-risk and backwards compatible, that way I can roll out fixes to legacy users without requiring them to perform code rewrites to upgrade (which many will never do).
